source/includes/add-new-app.md

```diff
@@ -0,0 +1,36 @@
+# Add a New App
+
+Tell us about an app we don't know about yet.
+We do our best to find all apps, but occasionally we miss one (there isn't a complete list for Google Play).
+
+### HTTPS Request
+
+`https://api.appmonsta.com/v1/stores/<store>/details/<app_id>.json`
+
+### Request Method: PUT
+
+### Request Parameters
+
+Parameter         | Required | Value
+----------------- | -------- | -----------
+**store**         | Yes      | `android` or `itunes`.
+**app_id**        | Yes      | the id of the app to add, e.g. com.rovio.angrybirds.
+
+### Response: 202 if successfully added
+
+Check right side shell for such example.
```

Comment:
shell in what context? since we have code samples for multiple languages? "Check the right side code section for an example"

Reply:
I meant just this curl example here...anyways, I'll put it under the text as it originally was:

![add_new_app_success](https://user-images.githubusercontent.com/3522071/47011560-ed128880-d14a-11e8-8f3b-1f56b979181d.png)

Reply:
Hm, OK. Later we should add code samples for the languages.